### ISSUE TITLE
Process got killed after Episode 17

### ISSUE THREAD

**Issue by ravishk1** [simoninithomas/Deep_reinforcement_learning_Course#62]
I was training the same code that you have provided, But it is getting killed after 17 episodes.
What can I infer from this? 

Model Saved
Episode: 1 Total reward: 105.0 Explore P: 0.9547 Training Loss 0.0046
Episode: 2 Total reward: 105.0 Explore P: 0.9408 Training Loss 0.0009
Episode: 3 Total reward: 80.0 Explore P: 0.9239 Training Loss 0.0023
Episode: 4 Total reward: 215.0 Explore P: 0.8979 Training Loss 6.2312
Episode: 5 Total reward: 460.0 Explore P: 0.8730 Training Loss 0.0272
Model Saved
Episode: 6 Total reward: 255.0 Explore P: 0.8475 Training Loss 0.0019
Episode: 7 Total reward: 315.0 Explore P: 0.8226 Training Loss 1.5508
Episode: 8 Total reward: 210.0 Explore P: 0.8038 Training Loss 0.0031
Episode: 9 Total reward: 365.0 Explore P: 0.7788 Training Loss 0.0116
Episode: 10 Total reward: 260.0 Explore P: 0.7584 Training Loss 0.0428
Model Saved
Episode: 11 Total reward: 510.0 Explore P: 0.7363 Training Loss 1.5399
Episode: 12 Total reward: 210.0 Explore P: 0.7190 Training Loss 0.0216
Episode: 13 Total reward: 215.0 Explore P: 0.7002 Training Loss 1.9142
Episode: 14 Total reward: 210.0 Explore P: 0.6834 Training Loss 0.3903
Episode: 15 Total reward: 415.0 Explore P: 0.6601 Training Loss 1.5385
Model Saved
Episode: 16 Total reward: 285.0 Explore P: 0.6416 Training Loss 0.0108
Episode: 17 Total reward: 180.0 Explore P: 0.6230 Training Loss 0.0026
Killed

**Comment by 2590477658** [comment]
Yes, I think it is because your memory is not enough, this kind of training does need a lot of resource
